src/docs/recommendation.md

```diff
@@ -26,9 +26,33 @@ G-Rec 2. Provide metadata to describe the crosswalks and individual mappings ([d
 
 #### Technical recommendation
 
-T-Rec 2.1 Use the minimum SSSOM metadata schema defined by the FAIR IMPACT project to document mappings and crosswalks ([discussion#63](https://github.com/mapping-commons/rda-fair-mappings/discussions/63))
-
-T-Rec 2.2 If the mappings are separated from the associated metadata, Include an additional metadata field containing the PID pointing to the mapping or crosswalk  ([discussion#64](https://github.com/mapping-commons/rda-fair-mappings/discussions/64))
+T-Rec 2.1 Use the minimum SSSOM metadata schema defined by the FAIR IMPACT project to document mappings and crosswalks ([discussion#63](https://github.com/mapping-commons/rda-fair-mappings/discussions/63)). We recommend to adopt the following SSSOM metadata elements as mandatory metadata to describe the mappings and crosswalks:
+
+- Metadata elements for **Findability** (SSSOM recommended properties in parentheses):
+   - Author ([author_id](https://mapping-commons.github.io/sssom/author_id/), [author_label](https://mapping-commons.github.io/sssom/author_label/))
+   - Creator ([creator_id](https://mapping-commons.github.io/sssom/creator_id/), [creator_label](https://mapping-commons.github.io/sssom/creator_label/))
+   - Reviewer ([reviewer_id](https://mapping-commons.github.io/sssom/reviewer_id/), [reviewer_label](https://mapping-commons.github.io/sssom/reviewer_label/))
+   - Date of creation of this version ([publication_date](https://mapping-commons.github.io/sssom/publication_date/))
+   - License ([license](https://mapping-commons.github.io/sssom/license/))
+   - Source and target SAs‘ IDs ([subject_source](https://mapping-commons.github.io/sssom/subject_source/), [object_source](https://mapping-commons.github.io/sssom/object_source/))
+   - Mapping method  (in SSSOM the mapping method is described by the [mapping_justification](https://mapping-commons.github.io/sssom/mapping_justification/))
+   - Software used to create the mapping (if any) ([mapping_tool_id](https://mapping-commons.github.io/sssom/mapping_tool_id/), [mapping_tool](https://mapping-commons.github.io/sssom/mapping_tool/), [mapping_tool_version](https://mapping-commons.github.io/sssom/mapping_tool_version/))
+   - Specific types of SA that are mapped ([?]())
```

Comment:
This does not exist in SSSOM - make issue?

Reply:
I think you should make an issue about this and the acceptable values would be informed by the discussions we have had in the group.

Reply:
@gouttegd points out: we have subject_type, object_type, this is I think all we need!

Reply:
@jkesanie says that for search / index it may be helpful to have these on mapping_set level.

Reply:
mapping_set_type vs subject_source_type

schema mapping vs schema ---> schema